src/Microsoft.Diagnostics.Monitoring.WebApi/DiagnosticServices.cs

```diff
@@ -26,14 +26,14 @@ internal sealed class DiagnosticServices : IDiagnosticServices
             _logger = logger;
         }
 
-        public async Task<IEnumerable<IProcessInfo>> GetProcessesAsync(DiagProcessFilter processFilterConfig, CancellationToken token)
+        public async Task<IEnumerable<IProcessInfo>> GetProcessesAsync(DiagProcessFilter? processFilterConfig, CancellationToken token)
         {
-            IEnumerable<IProcessInfo> processes = null;
+            IEnumerable<IProcessInfo>? processes = null;
```

Comment:
Could you assign this `Enumerable.Empty<IProcessInfo>()` so it doesn't have to be nullable?

Reply:
Sure, in general I've been trying to keep the code as close to original as possible given the amount of changes, and deferring cleanup to later PRs.

In this case though there's no risk of this change so I'll update it